CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Pending Additions
 
+- [#102](https://github.com/helmholtz-analytics/heat/pull/429) Add documentation for the generic reduce_op in Heat's core
```

Comment:
normally we have the numbers representative of the PR instead of the issue number. also the link is pointing to the wrong. for this PR it would be `[#521](https://github.com/helmholtz-analytics/heat/pull/521)`

Reply:
Oops, my bad thanks